### PR TITLE
Set max_fails to allow fallback to JPG to work

### PR DESCRIPTION
**Pull request by giorgiosironi**
Part of https://github.com/elifesciences/issues/issues/4728

### CODE REVIEW

**Comment by giorgiosironi** [comment]
No smoke tests whatsoever though. Anything we can add?

**Comment by lsh-0** [comment]
never knew about this [max_fails](https://nginx.org/en/docs/http/ngx_http_upstream_module.html#max_fails) healthchecking behaviour. 

I remember when nginx was brand new and tiny and everyone loved it because it didn't have all that old Apache baggage and arcane configuration.

**Comment by lsh-0** [comment]
as part of the fixed made to prod I switched the nginx config that the containerised iiif is using so `prod` isn't affected and re-applied your change to that file.

**Comment by lsh-0** [comment]
the failing `loris-ready` state is in `master` so this PR won't ever be entirely green.

taking a look at hooking those smoke tests in now

File: salt/iiif/config/lib-systemd-system-iiif-service.service [hunk 1]
```diff
@@ -6,10 +6,12 @@ Before=nginx.service
...
 [Service]
 TimeoutStartSec=0
-Restart=always
-
 WorkingDirectory=/opt/loris
-ExecStart=/usr/local/bin/docker-compose up
+ExecStart=/usr/local/bin/docker-compose up --detach
+ExecStop=/usr/local/bin/docker-compose down
+# https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=
+Type=oneshot
```
Comment (lsh-0) [inline salt/iiif/config/lib-systemd-system-iiif-service.service]: this is a bit mysterious to me but it works like a charm and is being recommended as a way for systemd and docker-compose to inter-operate.

**Comment by lsh-0** [comment]
@giorgiosironi , this is your PR so please merge if you're happy with it. I'm not sure what the state of end2end is at the moment but I imagine this will help. I'll continue debugging end2end on Thursday if it still needs it. If not, I'll upgrade the other environments and then prod.

Let me know if everything is ok but you still want me to hold off from upgrading `prod`

**Pull request merged**